keyvault/client/secrets/README.md

```diff
@@ -0,0 +1,295 @@
+# Azure Key Vault Secret client library for Java
+Azure Key Vault is a tool for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys or passwords. A vault is logical group of secrets.
+Secret client library allows you to securely store and tightly control access to tokens, passwords, API keys, and other secrets. The library offers operations to create, retrieve, update, delete, purge, backup, restore and and list the secrets.
+
+Use the secret client library to create and manage secrets.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation] | [Product documentation][azkeyvault_docs]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+- An existing [Azure Key Vault][azure_keyvault] : Note that this document shows how to create one.
+
+### Adding the package to your project
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-keyvault-secrets</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+</dependency>
+```
+
+### Create an Azure Key Vault.
+
+To create an Azure Key Vault you can use the Azure Portal or [Azure Keyvault CLI][azure_keyvault_cli].
+
+Create the Azure Key Vault:
+```Powershell
+az keyvault create --name <keyvault-name> --resource-group <resource-group-name> --location eastus
+```
+
+### Authenticate the client
```

Comment:
For this entire section Sameeksha had (here replacing 36-59  and moving the chunk about creating environment variables from Create Client section)
### Authenticate the client
In order to interact with the Key Vault service, you'll need to create an instance of the KeyVaultClient class. You need a **vault url**, **client secret credentials (client id, client secret, tenant id)** and a resource url to instantiate a client object.

 #### Create/Get credentials
Use the [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to create/get client secret credentials.

 * Create a service principal and configure its access to Azure resources:

     `az ad sp create-for-rbac -n <your-application-name> --skip-assignment`

* [STEP FOR SETTING ENVIRONMENT VARIABLES] -NEEDS TO INCLUDE STEPS AND DETAILED INSTRUCTIONS ON HOW TO SET THEM
* Grant the above mentioned application authorization to perform secret operations on the keyvault:

     `az keyvault set-policy --name <your-key-vault-name> --spn <your-service-principal-id> --secret-permissions <secret-permissions>`
    > --secret-permissions:
    > Space-separated list of secret permissions to assign. Accepted values: backup, delete, get, list, purge, recover, restore, set